src/sprites/rendered-target.js

```diff
@@ -943,6 +943,9 @@ class RenderedTarget extends Target {
                 other.drawableID, 0, StageLayering.SPRITE_LAYER, true);
             this.renderer.setDrawableOrder(this.drawableID, otherLayer, StageLayering.SPRITE_LAYER);
         }
+
+        const executionPosition = this.runtime.moveExecutable(other, 0);
```

Comment:
why not:
`const executionPosition = this.runtime.executableTargets.indexOf(other)` ?

You can even add a call to the math util clamp function if you want to ensure that the returned index is within the bounds of the list...